apps/schema/static/schema/components/devices/deviceTelemetry.yaml

```diff
@@ -0,0 +1,41 @@
+post:
+  summary: Add telemetry data
+  description: |
+    Associate telemetry data with a `Shipment`. The tracking payload must be signed and encoded as a [JWS](https://tools.ietf.org/html/rfc7515).
+    An active AXLE certificate must be used to sign this JWS, and the certificate ID must be included in the headers as `kid`.
+
+    A JSON Schema specification for the tracking data payload format is located at [schema.shipchain.io](http://schema.shipchain.io/1.2.1/tracking.json)
```

Comment:
Not tracking

Reply:
we also currently do not have the schema for telemetry data so I will have to add that.